### ISSUE TITLE
Connection unstability 

### ISSUE THREAD

**Issue by fhoshino** [zhukov/webogram#1967]
Constantly getting "waiting for network" between GMT+9 15:00 - 19:00, where as telegram desktop and phone does not have this connection problem.

**Comment by Aokromes** [comment]
maybe it's your is blocking it, try opera browser with private tab + vpn.

**Comment by fhoshino** [comment]
Nope, my ISP is not blocking it, also, have tried vpn already, it seems to be a server issue for me more than a client side one.
